packages/api/tests/endpoints/collections/get-collection.js

```diff
@@ -88,19 +114,23 @@ test('CUMULUS-912 GET with pathParameters and with an invalid access token retur
   assertions.isInvalidAccessTokenResponse(t, response);
 });
 
-test.todo('CUMULUS-912 GET with pathParameters and with an unauthorized user returns an unauthorized response');
-
-test.serial('GET returns an existing collection', async (t) => {
-  const stub = sinon.stub(EsCollection.prototype, 'getStats').returns([t.context.testCollection]);
+test('GET returns an existing collection', async (t) => {
+  const { testCollection } = t.context;
   const response = await request(app)
     .get(`/collections/${t.context.testCollection.name}/${t.context.testCollection.version}`)
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
-  const { name } = response.body;
-  stub.restore();
-  t.is(name, t.context.testCollection.name);
+  const expected = {
+    granuleId: testCollection.granule_id_validation_regex,
+    granuleIdExtraction: testCollection.granule_id_extraction_regex,
+    sampleFileName: testCollection.sample_file_name,
+    files: JSON.parse(testCollection.files),
+    name: testCollection.name,
+    version: testCollection.version,
+  };
+  t.like(response.body, expected);
```

Comment:
Is there some reason we can't do a deepEqual?

Reply:
I suppose not, if we were to pull in the auto generated keys from the response.body object.    I'd been in the habit of using .like for objects returning generated keys, relying on the underlying database code to make sure that the schema was correct, but perhaps that's not the best approach for these endpoint tests. 

How about: c2b3d9ed4

Reply:
c2b3d9ed4